2021-11-22-announcements.md

```diff
@@ -0,0 +1,37 @@
+# CFDE Announcements
+
+## Cross-Pollination Events
+Our next scheduled meeting will take place on **Tuesday, December 7th at 11am PST / 2pm EST**. December's meeting is scheduled to have presentations from R03s detailing their progress and experiences using CF data and tools.  Regular Cross-Pollination events take place on the first Tuesday of every month. Please check out the  [Cross-Pollination Schedule](https://docs.google.com/spreadsheets/d/1hQAeOLkivUZZnwZ_KxfGw3neezMaWbrPk9nnFiKfQGA/edit?usp=sharing) for more details.
+
+## IDG Digital Tool Fest
+IDG (Illuminating the Druggable Genome) is hosting the inaugural IDG Digital Tool Fest on **Tuesday, November 30th at 9am-12pm PT / 10am-1pm MT / 12-3pm ET.** It is a free, virtual meeting with 10-minute demonstration of IDG developed digital tools for exploring target-centric and drug-centric information. These tools offer users the ability to construct their own specialized queries to access information programmatically. 
+
+For more information and a listing of all the highlighted tools visit https://druggablegenome.net/IDG-Digital-Tool-Fest-Fall2021.
+To register, visit https://hsc-unm.zoom.us/meeting/register/tJEpfuGqqTooG9DC2YjXf5ah72tBXPcOGLaS.
+
+## CFDE Training Efforts
+For December we are planning a CFDE Portal Demo, an encore Intro to Unix session, and our popular Intro to AWS workshop.  Check out our [Events Calendar](https://www.nih-cfde.org/events/) for details and registration availability. 
```

Comment:
Maybe something like: 

We are planning to host a webinar and two workshops on the first three Wednesdays of the December at 10 am PST.  Check out our [Events Calendar](https://www.nih-cfde.org/events/) for details and registration availability. 

  * Dec. 1 Webinar: CFDE Portal Demo
  * Dec. 8 Workshop: Intro to Unix for Cloud Computing
  * Dec. 15 Workshop: Intro to AWS for Cloud Computing

Reply:
If the links are okay, I'm happy with this. Trying to conserve space...